### ISSUE TITLE
[E4] Duda sobre el caso especial de la negación de contraataque

### ISSUE THREAD

**Issue by martina-abello** [franciscogazitua/IIC2113-2024#209]
![imagen](https://github.com/user-attachments/assets/fa5c2a26-302d-46f5-83a0-11c41add8ebb)
 
por lo que veo en los tests, esto implica además que si la unidad que defiende tiene negación de contraataque, tampoco puede hacer follow up, aunque cumpla la condición de velocidad? 

gracias de antemano.

**Comment by cahinostroza** [comment]
Efectivamente, el follow-up de la unidad que ataca, desde los ojos de la unidad que defiende, se ve como un contrataque

**Comment by martina-abello** [comment]
vale pero esto entonces aplica sobre el atacante también? si el atacante tiene  negacion de counter y trata de hacer follow up también se lo niega?

**Comment by franciscogazitua** [comment]
Hola

Eso no pasaría. Contraatacar es la acción de contrarrestar el ataque del enemigo, lo que para efectos del juego considera el primer ataque y el Follow-Up que hace la unidad que se defiende. Por definición, la unidad que inicia el ataque no puede contra atacar, esto porque no responde a una ofensiva, sino que es el ofensor.

EDIT: Con el ultimo punto me refiero a que el primer ataque y el Follow-Up de quien se defiende no son contraataques, por lo que no pueden ser negados por las habilidades de negación de contraataque
